libs/payments/cart/src/lib/cart.utils.ts

```diff
@@ -17,6 +19,35 @@ export const handleEligibilityStatusMap = {
   [EligibilityStatus.INVALID]: CartEligibilityStatus.INVALID,
 };
 
+export const CartEligibilityDetailsMap: Record<
```

Comment:
Couple of recommendations here:
1. This should be camelCase, not PascalCase
2. Probably not a bad idea to export this as readonly